styleguide/components/callouts/callouts.stories.mdx

```diff
@@ -0,0 +1,94 @@
+import { Meta, Story, Preview, Source } from '@storybook/addon-docs/blocks';
+
+<Meta
+  title="Components/Callouts"
+  component={() => templateDefault}
+  decorators={[(Story) => `<div style="max-width: 800px">${Story()}</div>`]}
+/>
+
+# Callouts
+
+There are several different variants of the callout component:
+
+## Default
+
+import templateDefault from './_default.html.haml';
+import templateDefaultCode from '!!raw-loader!./_default.html.haml';
+
+<Preview>
+  <Story
+    name="Default callout"
+    parameters={{ docs: { source: { code: templateDefaultCode } } }}
```

Comment:
This little dance sets the code snippet. The default behaviour is to return the template string which resolves to a literal `() => templateDefault`. Wish this syntax was simpler but think It's worth it to be able to show good code-snippets.